code/modules/flufftext/Dreaming.dm

```diff
@@ -3,7 +3,7 @@ var/list/dreams = list(
 	"an ID card","a bottle","a familiar face","a crewmember","a toolbox","a security officer","the captain",
 	"voices from all around","deep space","a doctor","the engine","a traitor","an ally","darkness",
 	"light","a scientist","a monkey","a catastrophe","a loved one","a gun","warmth","freezing","the sun",
-	"a hat","the Luna","a ruined station","a planet","phoron","air","the medical bay","the bridge","blinking lights",
+	"a hat","the Luna","a ruined station","a planet","plasma","air","the medical bay","the bridge","blinking lights",
```

Comment:
Nope: https://github.com/ParadiseSS13/Paradise/commit/adef8d0f065bdc479c1c44d0c271ecc397223146 it used to be plasma; it got overwritten to phoron when Ponies updated from Bay, but didn't realize that they had changed the name.